packages/in_app_purchase/in_app_purchase_storekit/darwin/Classes/Protocols/FLTPaymentQueueProtocol.h

```diff
@@ -40,19 +40,17 @@ NS_ASSUME_NONNULL_BEGIN
 /// Will add completed transactions for the current user back to the queue to be re-completed. This
 /// version requires an identifier to the user's account.
 - (void)restoreCompletedTransactionsWithApplicationUsername:(nullable NSString *)username
-    API_AVAILABLE(ios(7.0), macos(10.9), watchos(6.2), visionos(1.0));
+    API_AVAILABLE(ios(7.0), macos(10.9));
 
 /// Call this method to have StoreKit present a sheet enabling the user to redeem codes provided by
 /// your app. Only for iOS.
-- (void)presentCodeRedemptionSheet API_AVAILABLE(ios(14.0), visionos(1.0))
-    API_UNAVAILABLE(tvos, macos, watchos);
+- (void)presentCodeRedemptionSheet API_AVAILABLE(ios(14.0))API_UNAVAILABLE(tvos, macos, watchos);
 
 /// If StoreKit has called your SKPaymentQueueDelegate's "paymentQueueShouldShowPriceConsent:"
 /// method and you returned NO, you can use this method to show the price consent UI at a later time
 /// that is more appropriate for your app. If there is no pending price consent, this method will do
 /// nothing.
-- (void)showPriceConsentIfNeeded API_AVAILABLE(ios(13.4), visionos(1.0))
-    API_UNAVAILABLE(tvos, macos, watchos);
+- (void)showPriceConsentIfNeeded API_AVAILABLE(ios(13.4))API_UNAVAILABLE(tvos, macos, watchos);
```

Comment:
In other places you removed `watchos`; we should either only remove `visionos` everywhere, since that's the problem, or we should leave `watchos` and `tvos` everywhere.

Reply:
thats fair. let me put them back.

Reply:
oh i see you edited your comment. Do you mean I should also remove watchos and tvos from API_UNAVAILABLE?

Reply:
Putting them back is fine; I edited my quote because I meant to say that we should be consistent one way or the other—either list watchos and tvos everywhere, or don't list them anywhere—but what I actually wrote was the same option two different phrased ways.